src/v0/destinations/marketo_bulk_upload/fileUpload.js

```diff
@@ -59,14 +72,14 @@ const getHeaderFields = (config, fieldSchemaNames, jobIds) => {
   columnFieldsMapping.forEach(colField => {
     if (fieldSchemaNames) {
       if (fieldSchemaNames && !fieldSchemaNames.includes(colField.to)) {
-        throw new CustomError(
+        throw new AbortedError(
           `The field ${colField.to} is not present in Marketo Field Schema. Aborting`,
           400,
           { successfulJobs: jobIds, unsuccessfulJobs: [] }
         );
       }
     } else {
-      throw new CustomError(`Marketo Field Schema is Empty. Aborting. `, 400, {
+      throw new AbortedError("Marketo Field Schema is Empty. Aborting", 400, {
```

Comment:
This is a marketo ecosystem setup problem (out side of rudderstack) shall we attribute as config @saikumarrs ?

Reply:
No customer has to fix it

Reply:
@mihir-4116 , let's fix this error type.